hoot-core/src/main/cpp/hoot/core/conflate/ConflateCmd.cpp

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Is this file (and it's associated .h) supposed to be here? It looks an awful lot like ConflateCumulativeCmd.cpp...